src/helpers/data.mjs

```diff
@@ -0,0 +1,35 @@
+import currentYear from "../config/year.mjs";
+import picks2018 from "../config/2018/picks.hockey.mjs";
+import results2018 from "../config/2018/results.hockey.mjs";
+import picks2019 from "../config/2019/picks.hockey.mjs";
+import results2019 from "../config/2019/results.hockey.mjs";
+import picks2020 from "../config/2020/picks.hockey.mjs";
+import results2020 from "../config/2020/results.hockey.mjs";
+import picks2021 from "../config/2021/picks.hockey.mjs";
+import results2021 from "../config/2021/results.hockey.mjs";
```

Comment:
This is really ugly. ESM does not allow for dynamically importing the files. I'm doing this right now to avoid refactoring something else in this PR. Once we land this, I'll refactor this so we are importing data more sensibly.